server/sonar-db-migration/src/main/java/org/sonar/server/platform/db/migration/version/v62/PopulateOrganizationUuidOfGroups.java

```diff
@@ -49,10 +49,10 @@ public void execute(Context context) throws SQLException {
     massUpdate.update("update groups set organization_uuid=?, updated_at=? where id=?");
     massUpdate.rowPluralName("groups");
     massUpdate.execute((row, update) -> {
-      long groupId = row.getLong(1);
+      int groupId = row.getInt(1);
```

Comment:
this is indeed correct (group id is an integer in DB) but this should either be a separate commit of the commit title should be updated